addons/stock/wizard/stock_return_picking.py

```diff
@@ -25,6 +25,9 @@ class stock_return_picking(osv.osv_memory):
     _columns = {
         'product_return_moves': fields.one2many('stock.return.picking.line', 'wizard_id', 'Moves'),
         'move_dest_exists': fields.boolean('Chained Move Exists', readonly=True, help="Technical field used to hide help tooltip if not needed"),
+        'parent_location_id': fields.many2one('stock.location'),
```

Comment:
I don't know where this idea for a parent_location_id comes from.  The location to return to, is either a return location or the source location of the original move itself.  (which can be a location zone so no need to specify a child_of)